features/transient.feature

```diff
@@ -465,7 +467,7 @@ Feature: Manage WordPress transient cache
       """
 
     When I run `wp transient list --network --format=csv`
-    Then STDOUT should be:
+    Then STDOUT should contain:
```

Comment:
Maybe doing something like this helps with the test failures:

```
    When I run `wp transient list --network --format=csv`
    Then STDOUT should contain:
      """
      foo4,bar4,false
      """
    And STDOUT should contain:
      """
      foo5,bar5,95649119999
      """
    And STDOUT should contain:
      """
      foo6,bar6,1321009871
      """
```

Reply:
That's a more robust way to fix the issue. I've updated the tests.